sklearn/datasets/tests/test_openml.py

```diff
@@ -745,12 +745,12 @@ def test_fetch_openml_emotions_pandas(monkeypatch):
 
     data_id = 40589
     target_column = [
-        "amazed.suprised",
+        "amazed.surprised",
         "happy.pleased",
         "relaxing.calm",
         "quiet.still",
         "sad.lonely",
-        "angry.aggresive",
+        "angry.aggressive",
```

Comment:
This is a problem: those typos are part of the original dataset hosted at openml.org. We should not fix those and we should find a way to silence false positives for this specific file without adding typos in the global `codespell_ignore_words.txt` file. Would it be possible to tell codespell to ignore some words in a specific file?

Reply:
Reverted to the openml.org typos, `aggresive` and `suprised` added to the list of false positives.